### PR TITLE
improved README

### PR DESCRIPTION
**Pull request by hrzn**
Made the README more public-facing. I moved all the notes about dev setups using Conda & pip to a private Quip page: https://unit8io.quip.com/EHR0A7vQSZWL/u8timeseries-dev-setup

Let's keep the README as simple and user-friendly as possible.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,38 +8,24 @@ The models can all be used in the same way, using `fit()` and `predict()` functi
 similar to scikit-learn. The library also makes it easy to backtest models,
 and combine the predictions of several models and external regressors.
 
-## install
-
-### using conda
```
Comment (TheMP) [inline README.md:13]: Should we also keep conda steps somewhere?

File: README.md [hunk 2]
```diff
@@ -69,17 +55,18 @@ plt.xlabel('Year')
...
+# Features
 Currently, the library contains the following features: 
 
 **Forecasting Models:** 
 * Exponential smoothing, 
 * ARIMA & auto-ARIMA,
 * Facebook Prophet,
-* Theta method, 
+* Theta method,
+* FFT (Fast Fourier Transform)
```
Comment (TheMP) [inline README.md:69]: This one is not actually released yet :)

File: README.md (hunk 1 above)
Comment (radujica) [reply README.md:13]: I see conda setup needs to be checked/rewritten with the addition of pytorch, so maybe a different PR?

issues include:
- it's torch on pip, pytorch on conda -.-
- pytorch actually has some very specific install steps depending on the GPU and on whether using pip vs conda

Comment (hrzn) [reply README.md:13]: If we have a general way to `conda install u8timeseries` then yes we should document it here. But for a future PR I think.

File: README.md (hunk 2 above)
Comment (hrzn) [reply README.md:69]: I know it will be very soon though, right @pennfranc ? :D

**Pull request merged**